modules/swagger-codegen/src/main/resources/csharp/Project.mustache

```diff
@@ -88,7 +88,10 @@
     {{/netStandard}}
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="**\*.cs"/>
+    <Compile Include="Api\**\*.cs"/>
```

Comment:
I also did a test and didn't experience any issue after the change related to how files are included.